packages/npm/openapi.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Script to generate OpenAPI typescript definitions for the Matrix APIs
+
+RELEASE="${npm_package_version%.*}"
+SPEC_DIR="v${RELEASE}"
+if [ "$RELEASE" == "0.0" ]; then
+  RELEASE="unstable"
+  SPEC_DIR="/unstable"
+fi
+
+BASE_URL="https://spec.matrix.org/$SPEC_DIR"
+
+for api in application-service client-server push-gateway server-server; do
+  FILE="$api-api.json"
```

Comment:
can we give this a better name? And a comment?

```suggestion
  # Combine the separate OpenAPI definition files into one document
  combined_doc="${api}-api.json"
```